src/webgpu/web_platform/external_texture/video.spec.ts

```diff
@@ -310,7 +337,7 @@ Tests that we can import an HTMLVideoElement into a GPUExternalTexture and use i
         kFormat,
         { x: 1, y: 0 },
         {
-          exp: new Uint8Array([0x00, 0xff, 0x00, 0xff]),
+          exp: kVideoExpectations[0].greenExpectation,
         }
```

Comment:
FYI, if you run into any issues with precision (needing a threshold because there is a range of possible results): #1121

Reply:
ack. It's consistent for me on my test platform, but I will not be surprised if a tolerance is needed when it is running on all platforms.

Reply:
Should we dry run this? I forget where the instructions are, but they've changed anyway, so I may as well try it out and document them.

Reply:
Turns out the dry run process hasn't been simplified the way I thought it might be yet, but here's one with the old method:
https://dawn-review.googlesource.com/c/dawn/+/91760

It has changed slightly so hopefully I got it right.

- check the current CTS revision in Dawn's DEPS file
- rebase the candidate change on that revision
- push to a branch in gpuweb/cts
- change Dawn's DEPS file to the revision of that branch
- wait for the branch to get mirrored to https://chromium.googlesource.com/external/github.com/gpuweb/cts/
- run `gen_ts_dep_lists.py`
- commit and upload a dawn PR
- run `dawn-*-deps-rel` tryjobs

Reply:
I'm not sure. I added a Mac tryjob to your CL, which is the one I'm worried about because of this comment: https://source.chromium.org/chromium/chromium/src/+/main:ui/gfx/color_space.h;l=91

I don't know if this enum is even used in the path we're using, and it may not have any impact anyway since we're testing primaries.

Reply:
Thanks. I only didn't add Mac at first because I wanted to make sure it was building at all (which failed the first time around).

That comment sounds like it applies only to the final display of the overlays, and not the data we would get by reading out of textures, but it's vague so hard to be sure. Good to test.

Reply:
(BTW, once those pass, feel free to land this PR.)

Reply:
Passed. Merging